### PR TITLE
Adds "edited" as a bot event

### PR DESCRIPTION
**Pull request by tcsc**
The review bot was not handling an "edited" event, and implicitly failing the check.

### CODE REVIEW

File: .github/workflows/ci/constants.go [hunk 1]
```diff
@@ -82,7 +82,10 @@ const (
 	// Created is an event type that is triggered when a pull request review is created.
 	Created = "created"
 
-	// AnyAuthor is the symbol used to get reviewers for external contributors/contrtibutors who
+	// Edit events are emitted when a PR Title or Description is edited
```
Comment (codingllama) [inline .github/workflows/ci/constants.go:85]: nit: final stop

```suggestion
	// Edit events are emitted when a PR Title or Description is edited.
```

**Review by codingllama — APPROVED** [review]

**Review by codingllama — APPROVED** [review]

**Comment by russjones** [comment]
@quinqu Do we need to support the edited event in the new bot?

**Comment by quinqu** [comment]
@russjones I'm not sure, @tcsc can you link the failing check?

**Comment by quinqu** [comment]
@russjones Just found out, it will not work with your new bot code

**Pull request closed without merging**